package.json

```diff
@@ -138,7 +138,8 @@
       "test",
       "fixture",
       "CustomEvent",
-      "ANSWERS"
+      "ANSWERS",
+      "requestAnimationFrame"
```

Comment:
do we still use requestAnimationFrame in this PR?

Reply:
ah, good catch. I ended up removing it from this PR